app/apikey.go

```diff
@@ -258,9 +263,17 @@ func NewAPIKeyCommand(getAPIKeyClientFn GetAPIKeyClientFn) (CommandOut, error) {
 					Name:    "list",
 					Usage:   "List apikeys",
 					Aliases: []string{"l"},
-					Flags:   []cli.Flag{},
+					Flags: []cli.Flag{
+						&cli.StringFlag{
+							Name:    ownerIDFlagName,
+							Usage:   "The owner id of the API Keys to list",
+							Aliases: []string{"o"},
```

Comment:
just double checking, this flag is optional.

not providing the owner id implies you are listing your own api keys.

Reply:
Based on the current usage, it'll return your own api keys, or if you're a global admin, all api keys.

Reply:
And yes, this flag is always optional